packages/cosmos_lint/lib/analysis_options_package.yaml

```diff
@@ -0,0 +1,228 @@
+include: package:lint/analysis_options.yaml
+
+analyzer:
```

Comment:
seems its a duplication of `lib/analysis_options.yaml`, can't those be reused?

Reply:
This is Incase we have different listing for projects and packages but this can be combined